css/_builders-challenge.scss

```diff
@@ -0,0 +1,112 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$image-path: '../../img';
+$font-path: '../../fonts';
+
+.c-builders-hero {
+    text-align: center;
+    background-image: url("/img/builders-challenge/hero-sm.png");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+
+    h1 {
+        @include font-size(56px);
+    }
+
+    h2 {
+        @include font-size(28px);
+    }
+
+    .mzp-l-content {
+        padding-top: 96px;
+        padding-bottom: 96px;
+    }
+
+    @media #{$mq-sm} {
+        background-image: url("/img/builders-challenge/hero-md.png");
+    }
+
+    @media #{$mq-lg} {
+        background-image: url("/img/builders-challenge/hero-lg.png")
+    }
+
+    @media #{$mq-xl} {
+        background-image: url("/img/builders-challenge/hero-xl.png")
+    }
+
+    @media (min-width: 1440px) {
+        background-image: url("/img/builders-challenge/hero-2x.png")
+    }
+}
+
+// overrides from protocol newsletter to have single column layout
+.c-builders-newsletter {
+    padding-bottom: $spacing-xl;
+
+    h3 {
+        text-align: center;
+    }
+
+    .mzp-c-newsletter {
+        padding-left: 0;
+        padding-right: 0;
+        grid-template-columns: 1fr;
+
+        .builders-form {
+            grid-column: 1;
+            margin: 0 auto;
+            max-width: 510px;
+            width: 100%;
+
+            .interests {
```

Comment:
**nit (non-blocking)**: prefer lower selector specificity as much as possible. This could be `.c-builders-newsletter .interests` or, with a class rename `.builders-form-interests`

<img width="688" alt="minimal-specificity" src="https://user-images.githubusercontent.com/19650432/224670778-15405afb-9188-4237-aa59-a8f0c23186de.png">